### DISCUSSION TITLE
CKEditor replacement

### DISCUSSION THREAD

**Discussion by ghost** [HuubMons/HuMo-genealogy#28] [Ideas]
What we can choose, discussion here

**Comment by ghost** [comment]
https://github.com/quilljs/quill
https://github.com/summernote/summernote

**Comment by HuubMons** [comment]
I tried Summernote in the past.
Possible problem: this application saves images as base64 images into the database (not saved as image file).

I will check quill, thanks!

**Reply by ghost** [reply]
> I tried Summernote in the past. Possible problem: this application saves images as base64 images into the database (not saved as image file).
> 
> I will check quill, thanks!

So bad.

**Comment by ghost** [comment]
Wysiwyg is used for what in humogen?

**Comment by HuubMons** [comment]
It's possible to add your own pages in HuMo-genealy. There is a build in CMS system.

Admin - Control -> CMS Own pages

**Comment by ghost** [comment]
https://github.com/codex-team/editor.js

**Comment by HuubMons** [comment]
https://www.tiny.cloud/get-tiny/

**Reply by ghost** [reply]
Tinymce is a great and popular editor like ckeditor. I used it a lot in the past.

**Comment by HuubMons** [comment]
CKEDitor is replaced with TinyMCE
